server/database/queries/eventPost.js

```diff
@@ -1,13 +1,13 @@
-const dbconniction = require('../db_connection')
+const dbConniction = require('../db_connection')
 
 const postEvent = (data) => {
   const sql = {
     text: 'INSERT INTO events (event_title ,event_description ,event_note,event_date,user_id,room_id ,event_author) VALUES ($1,$2,$3,$4,$5,$6,$7)',
     values: [data.title, data.description, data.notes, data.date, data.userId, data.roomId, data.name]
   }
-  dbconniction.query(sql, (err, result) => {
-    if (err) throw err
-    else console.log('a new event added successfuly')
-  })
+  dbConniction.query(sql)
+    .then(response => console.log(response.rows))
```

Comment:
Delete the promise at all from the query, use controller as a promise and don't send anything here, in the controller you used   res.send() so put this in resolved and message